qiskit_ibm_runtime/ibm_backend.py

```diff
@@ -357,6 +358,17 @@ def target_history(self, datetime: Optional[python_datetime] = None) -> Target:
         self._convert_to_target(refresh=True)
         return self._target
 
+    def refresh(self) -> None:
+        """Retrieve the newest backend configuration and refresh the current backend target."""
+        if config := configuration_from_server_data(
+            raw_config=self._service._api_client.backend_configuration(self.name, refresh=True),
+            instance=self._instance,
+        ):
+            self._configuration = config
+        self._get_properties(datetime=python_datetime.now())
+        self._get_defaults()
```

Comment:
Looks like `_get_defaults` also uses cached data if it already exists.

https://github.com/Qiskit/qiskit-ibm-runtime/blob/f8323dc45d1affc189b0b9e5b0684b92c1bb18e5/qiskit_ibm_runtime/ibm_backend.py#L251-L253